packages/data-layer/src/queries.ts

```diff
@@ -31,6 +30,37 @@ export const getProgramsByUser = gql`
   }
 `;
 
+/**
+ * Get all the profiles that a user is a part of in allo v2
+ * @param $alloVersion - The version of Allo
+ * @param $address - The address of the user
+ * @param $chainId - The network ID of the chain
+ *
+ * @returns The programs
+ */
+export const getProfilesByUser = gql`
```

Comment:
We have one for projects below called getProjectsByAddress that is very similar - I think our naming is starting to bleed about.